doc/source/theory/dem/dem.rst

```diff
@@ -82,26 +82,26 @@ The relative velocities calculated to allow updating the tangential overlap are
      - Linear model definitions
      - Nonlinear viscoelastic model definitions
    * - Normal spring constant
-     - :math:`k_n = \frac{16}{15}\sqrt{R_{e}}Y_{e}\left(\frac{15m_{e}V^2}{16\sqrt{R_{e}}Y_{e}}\right)^{0.2}`
+     - :math:`k_n = 1.2024(m^{0.5}E)Y_{e}^{2}R_{e}\nu_{e})^{0.4}`
      - :math:`k_n = \frac{4}{3}Y_{e}\sqrt{R_{e}\delta_n}`
    * - Tangential spring constant
      - :math:`\eta_n = \sqrt{\frac{4m_{e}k_n}{1+\left(\frac{\pi}{\ln{e}}\right)^2}}`
```

Comment:
```suggestion
     - :math:`\eta_n = -\sqrt{\frac{4m_{e}k_n}{1+\left(\frac{\pi}{\ln{e}}\right)^2}}`
```

Reply:
Since I've gone back to the original formula, the minus sign is no longer necessary.

Reply:
eta_n for the linear model is written similarly to the non-linear form. (with the beta parameter)